pages/about.tsx

```diff
@@ -19,14 +21,17 @@ const SectionWrapper: React.FC<SectionWrapperProps> = ({
 }) => {
   return (
     <>
-      <Head>
-        <meta property="og:type" content="website" />
-        <meta property="title" content="EVM Codes - About the EVM" />
-        <meta
-          name="description"
-          content="Demystifying what makes the Ethereum Virtual Machine work."
-        />
-      </Head>
+      <html lang="en"></html>
+      <React.Fragment>
+        <Head>
+          <title> EVM Codes - About the EVM </title>
+          <h1>
+            Understanding the Ethereum Virtual Machine (or EVM) is understanding
```

Comment:
I don't love this language, it's too much like a fortune cookie a proposal "Understand the relationship between opcode instructions, gas costs, storage, and the execution environment to better understand the EVM" or something